### DISCUSSION TITLE
Counting to 100 Million - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#14] [General]
# Counting to 100 Million - Alan Pope's blog

This article previously appeared on listed.to. I’ve moved it here to consolidate my blogging
About 10-15 years ago, back in the heady days of Hampshire Linux User Group, we had a Wiki. It ran a heavily patched version of UseModWiki that we’d modified to add anti-spam and anti-abuse protection. We’d affectionately called it “AbuseMod”. It’s still kinda there, but I don’t think the content is ever touched. We used it to co-ordinate meetings, take notes, and some other fun sillyness.

https://popey.com/blog/2020/12/counting-to-100-million/

**Comment by filviu** [comment]
```bash
$ time perl -e 'for($i=0;$i<1e8;$i++) { }'

real    0m1.214s
user    0m1.201s
sys     0m0.005s
```

Fastest I could get it (and no, I did not spend more time than I should running it using ansible ad-hoc on the many machines I herd). This was the fastest, on an  i5-12500. I first smuggly went for the 80 core ARM machine then I realised it's not multicore :)
